xbmc/addons/settings/AddonSettings.cpp

```diff
@@ -684,7 +677,7 @@ std::shared_ptr<CSettingCategory> CAddonSettings::ParseOldCategoryElement(uint32
 bool CAddonSettings::InitializeFromOldSettingDefinitions(const CXBMCTinyXML& doc)
 {
   m_logger->debug("trying to load setting definitions from old format...");
-
+  
```

Comment:
Please remove the white space at the end of the line.

Reply:
oops, my editor adds that on new lines.
Removed now

Reply:
@matthuisman any chance you want to quickly fix this up so we can get it merged when you have some time.

Reply:
The whitespace? It is already fixed :)

Reply:
@fuzzard oops - your right - now should be fixed 👍

Reply:
cheers mate. ill merge it when jenkins finishes